lib/data_update_scripts/20210430043750_remove_site_config_secondary_logo.rb

```diff
@@ -0,0 +1,7 @@
+module DataUpdateScripts
+  class RemoveSiteConfigSecondaryLogo
+    def run
+      SiteConfig.destroy_by(var: "secondary_logo_url")
```

Comment:
I decided to also remove the variable from the database, no need to keep it around if we're not using it. I can remove this or do it in a later PR if people prefer that but I thought it's unlikely that we'll change our mind and bring `secondary_logo` back any time soon.